tracer/src/Datadog.Trace/Debugger/ExceptionAutoInstrumentation/ExceptionTrackManager.cs

```diff
@@ -215,30 +226,30 @@ private static void ProcessException(Exception exception, ErrorOriginKind errorO
 
                     // Check if we failed to instrument all the probes.
 
-                    if (trackedExceptionCase.ExceptionCase.Probes.Any(p => p.ProbeStatus == Status.RECEIVED))
+                    if (trackedExceptionCase.ExceptionCase.Probes.Any(p => p.ProbeStatus == Status.RECEIVED || p.ProbeStatus == Status.INSTALLED))
                     {
                         // Determine if there are any errored probe statuses by P/Invoking the native for RECEIVED probes.
```

Comment:
NIT: might be worth updating the comment now that INSTALLED probes are considered as well

Reply:
Done in 753b806c79ee4ed635377cbe17dec51ba5641c29.